honeybee/_shadeparent.py

```diff
@@ -0,0 +1,227 @@
+"""Base class for all geometry objects that can host shades."""
+from .shade import Shade
+
+
+class _ShadeParent(object):
```

Comment:
Hm.  I could see why you might want to change it but the reality is that the relationship between a Room and its Shades or an Aperture and its Shades is a parent-child object relationship (just like the relationship between Face and Aperture).  Shades can be added/removed from an Aperture in the same way that an Aperture can be added/removed from a Face.
On the other hand, the host/parasite relationship as we have defined it in the rest of the library is essentially an inseparable one from the perspective of the parasite.  The FaceEnergyProperties can only exist when they are attached to a host Face and they cannot be separated from that Face.  So "Parent" is definitely a more appropriate word than "Host" here.  If you want to offer up another naming suggestion, I will consider it but I don't think we should consider making the relationship between shades a true host/parasite relationship, particularly because we want to allow people to make shades with no particular parent.

Reply:
Thankfully `BaseWithShade` was a much better choice and we can close this out.